src/main/java/entities/Task.java

```diff
@@ -25,7 +25,7 @@ public class Task {
     /**
      * A list of the associated tags of this Task
      */
-    ArrayList<Tag> tags;
+//    ArrayList<Tag> tags;
```

Comment:
Later when you uncomment this, you should change this to use a List instead of an ArrayList